pilota-build/src/parser/protobuf/mod.rs

```diff
@@ -427,6 +430,10 @@ impl Parser for ProtobufParser {
     fn parse(self) -> super::ParseResult {
         let descriptors = self.inner.parse_and_typecheck().unwrap().file_descriptors;
 
+        self.files.iter().for_each(|f| {
```

Comment:
`self.files` doesn't contains the files depended  by `input`

Reply:
Still get confused